x/token/types/types_test.go

```diff
@@ -0,0 +1,116 @@
+package types
```

Comment:
@youngjoon-lee 
How about changing the package name to `package types_test`?
The variables declared in the test can be shared to the actual service, so it would be good to be separated.

Reply:
Of course, Can not tested private method...

Reply:
Ya. Good point.
It's a good practice to define `*_test` packages for black-box tests, but then we cannot test private functions.
Here, we cannot use the `types_test` package because we're testing private `validate*` functions ㅜㅠ.

Reply:
Oh, I understand.